CHANGELOG.md

```diff
@@ -3,24 +3,36 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
-## [Unreleased]
+
+## [0.14.0] - (final release date unknown)
 
 ### Added
 
 - Add network timeout setting - [#324](https://github.com/owncloud/user_ldap/issues/324)
+- Log bind errors [#436](https://github.com/owncloud/user_ldap/pull/436)
+- Reuse existing LDAP accounts if available [#165](https://github.com/owncloud/user_ldap/pull/165)
 
 ### Changed
 
 - Allow avatars to be changed by users if not provided by LDAP - [#188](https://github.com/owncloud/user_ldap/issues/188)
 - Remove PHP 5.6 support - [#388](https://github.com/owncloud/user_ldap/issues/388)
-- LDAP configuration is now stored JSON encoded - [#393](https://github.com/owncloud/user_ldap/issues/393)
 - Clean up Application initialization code - [#396](https://github.com/owncloud/user_ldap/issues/396)
 - Remove unused use statements - [#399](https://github.com/owncloud/user_ldap/issues/399) [#400](https://github.com/owncloud/user_ldap/issues/400)
+- Bump bamarni/composer-bin-plugin from 1.2.0 to 1.3.0 [#401](https://github.com/owncloud/user_ldap/pull/401)
```

Comment:
This is a  dependency for tests only. Do we want/need to list it in the changelog?

Reply:
Deleted from changelog. Thanks for reviewing